roles/openshift_metrics/tasks/generate_certificates.yaml

```diff
@@ -119,14 +117,17 @@
 - name: generate password for htpasswd file for hawkular metrics
   shell: cat /dev/urandom | tr -dc _A-Z-a-z-0-9 | head -c15
   register: hawkular_metrics_password
+- name: generate password for hawkular metrics jgroups
```

Comment:
Possibly a later refactoring to keep in mind that may be of use: 
https://github.com/ewolinetz/openshift-ansible/blob/logging_deployer_tasks/roles/openshift_logging/filter_plugins/generators.py#L6  

I added this for generating the variable bit of our DC names.  I'm not sure if there is any advantage or desire to try to use modules and filters over shells and commands